anvil/plot.py

```diff
@@ -0,0 +1,317 @@
+from math import ceil, floor, log10, fabs
+
+import numpy as np
+import pandas as pd
```

Comment:
perhaps plot.py should just include plots and tables could go elsewhere? like table.py?

Reply:
makes sense